pom.xml

```diff
@@ -110,35 +110,40 @@
                 <!-- the Sling instance is provisioned from the model in src/main/provisioning/model.txt -->
                 <groupId>org.apache.sling</groupId>
                 <artifactId>slingstart-maven-plugin</artifactId>
-                <version>1.7.0</version>
+                <version>1.7.16</version>
                 <extensions>true</extensions>
                 <executions>
                     <execution>
                         <id>prepare-launchpad-package</id>
+                        <phase>pre-integration-test</phase>
                         <goals>
                             <goal>prepare-package</goal>
                         </goals>
                     </execution>
                     <execution>
                         <id>build-launchpad-package</id>
+                        <phase>pre-integration-test</phase>
                         <goals>
                             <goal>package</goal>
                         </goals>
                     </execution>
                     <execution>
                          <id>start-container-before-IT</id>
+                        <phase>pre-integration-test</phase>
```

Comment:
this is the default phase for start

Reply:
Yeah, I added that mostly for consistency as I need the other executions moved - but if you think it is clear enough without that I'm fine with removing it.